lib/shimmer.js

```diff
@@ -97,7 +98,8 @@ function applyDebugState(shimmer, shim, nodule) {
  */
 function instrument(agent, nodule, moduleName, resolvedName) {
   var instrumentation = shimmer.registeredInstrumentations[moduleName]
-  if (properties.hasOwn(nodule, '__NR_instrumented')) {
+  if (properties.hasOwn(nodule, '__NR_instrumented')
+      || properties.hasOwn(nodule, '__NR_instrumented_errored')) {
```

Comment:
Minor: would be nice for the log message to indicate it may have skipped the errored state.

Reply:
This would technically log (assuming logging is enabled) in the catch statement prior, right after we set `__NR_instrumented_errored ` to true. In that case, would you still want a log statement added here?

Reply:
Specifically, here: https://github.com/newrelic/node-newrelic/pull/450/files#diff-a73cf9db6d0ef10539108450179052caR138

Reply:
Or, are you asking me to reword the log?

Reply:
If you want it reworded, would you want two separate logs? One for skipping already instrumented -- and one for prior instrumentation failed? -- or would you rather me just include both in the same log message?

Reply:
Was thinking would be nice to acknowledge here, too. For the case where someone (or us accidentally in the future) tries over and over again as another hint of what is happening without deep-diving. We'd likely track down where the message was written anyways, which is why I think it is a minor issue.

I think it would be adequate to add some additional language "Already instrumented or failed to instrument...". If wanted to be super helpful could output which sate or log different messages.

But also, I think the PR is good enough to land if you do not wish to take that on.

Reply:
No worries dude, I can change this real quick 👍 it's been wonderful working with you all on this and I appreciate all the feedback